examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,650 @@
+      "  },\n",
+      "  \"cuisine\": [\n",
+      "    {\n",
+      "      \"name\": \"Sushi\",\n",
+      "      \"description\": \"Vinegared rice with raw fish\"\n",
+      "    },\n",
+      "    {\n",
+      "      \"name\": \"Ramen\",\n",
+      "      \"description\": \"Japanese noodle soup\"\n",
+      "    },\n",
+      "    {\n",
+      "      \"name\": \"Tempura\",\n",
+      "      \"description\": \"Batter-fried seafood and vegetables\"\n",
+      "    }\n",
+      "  ]\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\", json_mode=True)\n",
+    "@prompt_template(\"Provide information about {city} in JSON format\")\n",
+    "def city_info(city: str): ...\n",
+    "\n",
+    "\n",
+    "response = city_info(\"Tokyo\")\n",
+    "print(response.content)  # This will be a JSON-formatted string"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "JSON mode is beneficial for:\n",
+    "- Ensuring structured outputs from LLMs\n",
+    "- Easy integration with data processing pipelines\n",
+    "- Creating APIs that return JSON data\n",
+    "\n",
+    "Note that not all providers have an explicit JSON mode. For those providers, we attempt to instruct the model to provide JSON; however, there is no guarantee that it will output only JSON (it may start with some text like \"Here is the JSON: ...\"). This is where Output Parsers can be useful.\n",
+    "\n",
+    "It's also worth noting that you can combine `json_mode=True` with `response_model` to automatically parse the JSON output into a Pydantic model. This provides both the structure of JSON mode and the type safety of response models. Here's an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-10T01:42:11.605827Z",
+     "start_time": "2024-09-10T01:42:10.205896Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Name: Tokyo, Population: 13929286, Country: Japan\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pydantic import BaseModel\n",
+    "\n",
+    "from mirascope.core import openai, prompt_template\n",
+    "\n",
+    "\n",
+    "class CityInfo(BaseModel):\n",
+    "    name: str\n",
+    "    population: int\n",
+    "    country: str\n",
+    "\n",
+    "\n",
+    "@openai.call(\"gpt-4o-mini\", json_mode=True, response_model=CityInfo)\n",
+    "@prompt_template(\"Provide information about {city} in JSON format\")\n",
+    "def city_info(city: str): ...\n",
+    "\n",
+    "\n",
+    "response = city_info(\"Tokyo\")\n",
+    "print(\n",
+    "    f\"Name: {response.name}, Population: {response.population}, Country: {response.country}\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This approach combines the benefits of JSON mode with the type safety and validation of response models.\n",
```

Comment:
nit: this is essentially duplicated from the text before the code snippet, either remove it there or remove it here?